App/automotive/src/main/java/com/amos/infotaimos/view/LandingPageTile.kt

```diff
@@ -0,0 +1,26 @@
+package com.amos.infotaimos.view
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import androidx.cardview.widget.CardView
+import com.amos.infotaimos.R
+import com.amos.infotaimos.databinding.LandingPageTileBinding
+
+class LandingPageTile(context: Context, attrs: AttributeSet) : CardView(context, attrs) {
+
+    init {
+        var binding = LandingPageTileBinding.inflate(LayoutInflater.from(context))
+        addView(binding.root)
+        val attributes = context.obtainStyledAttributes(attrs, R.styleable.LandingPageTile)
+        binding.TileImage.setImageDrawable(attributes.getDrawable(R.styleable.LandingPageTile_tileIcon))
+        binding.TileText.text = attributes.getText(R.styleable.LandingPageTile_tileText)
+        binding.tileLayout.setBackgroundColor(
+            attributes.getColor(
+                R.styleable.LandingPageTile_tileColor,
+                50000
```

Comment:
whats with the number instead of a theme identifier?

Reply:
Eigentlich sollte der Wert nie gecalled werden, weil man immer ne Farbe provided, aber die Methode nervt sonst kann da gerne aber auch noch ne Color resource einfügen stattdessen

Reply:
Müsste ja der Wert fürs Theme sein also warum nicht einfach R.style.Theme_InfotAiMOS einfügen :D